samples/sample_common/src/vaapi_device.cpp

```diff
@@ -397,7 +397,7 @@ mfxStatus CVAAPIDeviceWayland::RenderFrame(mfxFrameSurface1 * pSurface, mfxFrame
             return mfx_res;
     }
 
-    m_Wayland->RenderBuffer(m_wl_buffer, pSurface->Info.CropW, pSurface->Info.CropH);
+    m_Wayland->RenderBuffer(m_wl_buffer, pSurface->Info.CropW, pSurface->Info.CropH, pSurface);
```

Comment:
If we pass the whole pSurface, why do we need to pass CropW and CropH separately?

Reply:
I can clean up it. Just direct passing only pSurface. CropW and CropH settle in RenderBuffer function.

Reply:
Thanks